packages/react/src/components/info-token/tests/InfoToken.spec.tsx

```diff
@@ -1,184 +1,184 @@
 /* eslint-disable testing-library/no-container */
-import {render, screen} from '@test-utils';
+import {render, screen, waitFor} from '@test-utils';
 import * as React from 'react';
 
 import {InfoToken, InfoTokenMode, InfoTokenSize, InfoTokenType} from '../InfoToken';
 
 describe('InfoToken', () => {
-    it('renders the info icon if the specified size and type are Small and Information', () => {
+    it('renders the info icon if the specified size and type are Small and Information', async () => {
         render(<InfoToken type={InfoTokenType.Information} mode={InfoTokenMode.Stroked} size={InfoTokenSize.Small} />);
 
-        const smallToken = screen.getByRole('img', {name: 'info'});
+        const smallToken = await waitFor(() => screen.findByRole('img', {name: 'info'}));
```

Comment:
findByRole is already async, no need to use the waitFor on top of it. (here and in many other places)

```suggestion
        const smallToken = await screen.findByRole('img', {name: 'info'});
```